src/components/Permissions/AccessRightsSection.jsx

```diff
@@ -30,11 +33,27 @@ const AccessRightsSection = ({
   return (
     <NavigationListSection>
       <ListItem>
-        <Typography variant="h5">
-          {!isRemoteDoctypes
-            ? t('Permissions.limited_right_access')
-            : t('Permissions.exit_rights')}
-        </Typography>
+        <ListItemIcon>
+          <Icon
+            icon={isRemoteDoctypes ? CozyReleaseIcon : CozyLockIcon}
+            size={largeSize}
+          />
+        </ListItemIcon>
+        <ListItemText
+          primary={
+            <Typography type="body2" style={{ fontWeight: 'bold' }}>
+              {!isRemoteDoctypes
+                ? t('Permissions.limited_right_access')
+                : t('Permissions.exit_rights')}
+            </Typography>
+          }
+          secondary={
+            !isRemoteDoctypes
+              ? t('Permissions.limited_right_access_secondary')
+              : t('Permissions.exit_rights_secondary')
+          }
+          primaryTypographyProps={{ color: 'red' }}
```

Comment:
Why is this necessary? I don't see any red in the screenshot (and I'm not sure we should use hardcoded colors)

Reply:
my bad, this was for testing and was not working. I remove it